.github/workflows/authors-and-third-party-notices.yaml

```diff
@@ -23,14 +23,14 @@ jobs:
           fetch-depth: '0'
 
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
-          node-version: ^16.20.2
+          node-version: ^18.19.1
```

Comment:
We use exact, not range

Reply:
Ahh wow, I didn't check that. Thanks - I will check and fix.

Reply:
wasn't it `^16.20.2` before which is a range?

Reply:
It's not a range in evergreen

Reply:
Oh you mean in evergreen. Forgot this is to make it the same as there and not just to bump it. Carry on, then ;)